src/ActionModal/components/ActionModal.vue

```diff
@@ -463,8 +463,10 @@ export default {
         this.gasEstimateLoaded = true
         return fee
       }
-
-      this.gasEstimateLoaded = true
+      // in development we don't need to worry about fees. Next button won't be disabled
```

Comment:
if we don't need to worry about fees, why not just set it to true always?

Reply:
because then users will have the "Next" button always enabled. But I see that you saw that